lib/cosmos/gui/line_graph/line_graph.rb

```diff
@@ -369,9 +375,27 @@ def clear_lines
     end # def clear_lines
 
     # Adds a line to the graph - Afterwards the graph is ready to be drawn
-    def add_line(legend_text, y, x = nil, y_labels = nil, x_labels = nil, y_states = nil, x_states = nil, color = 'blue', axis = :LEFT, max_points_plotted = nil)
+    #  color = 'auto' automatically determines color from index based lookup
+    def add_line(legend_text, y, x = nil, y_labels = nil, x_labels = nil, y_states = nil, x_states = nil, color = 'auto', axis = :LEFT, max_points_plotted = nil)
       @unix_epoch_x_values = false unless x
 
+      # if color specified as auto, do lookup
+      if (color == 'auto')
```

Comment:
I believe he added it so that you could just add lines from line_graph_script without having to specify different colors manually.  I used to just default to blue.